abracadabra/src/writer/persistency_manager.cc

```diff
@@ -0,0 +1,64 @@
+#include "persistency_manager.hh"
+#include "g4-mandatory/trajectory.hh"
+
+
+persistency_manager::persistency_manager() {
+    G4String hdf5file = "test_file.h5";
```

Comment:
The `hdf5file` intermediate name doesn't improve the legibility of the code.

Reply:
Removed.